Tests/Data/Parabolic/T/3D_3BHEs_array/bcs_tespy.py

```diff
@@ -1,5 +1,5 @@
 ###
-# Copyright(c) 2012 - 2019, OpenGeoSys Community(http://www.opengeosys.org)
+# Copyright(c) 2012 - 2020, OpenGeoSys Community(http://www.opengeosys.org)
```

Comment:
```suggestion
# Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
```
Otherwise you have to update it every year forever... ;)

Reply:
Thanks for the hit. fixed in 539026853cfea9199624505f6da4779f9d909fec.

Reply:
Thanks for the hit. Fixed in 539026853cfea9199624505f6da4779f9d909fec.